engine/src/main/java/org/pentaho/di/www/StatusServletUtils.java

```diff
@@ -89,9 +92,9 @@ public static String getPentahoStyles() {
       // log here
     }
 
-    sb.append( "<link rel=\"stylesheet\" type=\"text/css\" href=\"/pentaho/content/common-ui/resources/themes/" + themeName + "/" + themeCss + "\"/>" );
-    sb.append( "<link rel=\"stylesheet\" type=\"text/css\" href=\"/pentaho/mantle/themes/" + themeName + "/" + mantleThemeCss + "\"/>" );
-    sb.append( "<link rel=\"stylesheet\" type=\"text/css\" href=\"/pentaho/mantle/MantleStyle.css\"/>" );
+    sb.append( "<link rel=\"stylesheet\" type=\"text/css\" href=\"" + root + "/content/common-ui/resources/themes/" + themeName + "/" + themeCss + "\"/>" );
```

Comment:
Please use multiple `.appen()` calls instead of concatenating string and then appending the result to the buffer.

Reply:
Let me know if the change added makes sense. Thanks.